package.json

```diff
@@ -54,50 +54,45 @@
     "storybook": "start-storybook"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.0.0",
     "@quilted/react-testing": "^0.1.4",
     "@quilted/sewing-kit-plugins": "^0.1.26",
     "@sewing-kit/cli": "^0.1.6",
     "@shopify/eslint-plugin": "^37.0.1",
     "@shopify/jest-dom-mocks": "^2.9.1",
+    "@shopify/webpack-ignore-typescript-export-warnings-plugin": "^1.0.4",
     "@storybook/addon-a11y": "^6.2.7",
     "@storybook/addon-controls": "6.1.18",
     "@storybook/addon-knobs": "^6.1.0",
     "@storybook/preact": "^v6.2.0-rc.3",
+    "@svgr/webpack": "^5.1.0",
     "@types/css-font-loading-module": "0.0.4",
     "@types/faker": "^4.1.12",
     "@types/react": "^16.9.20",
     "@types/react-dom": "^16.9.8",
     "babel-jest": "^26.6.3",
     "babel-loader": "^8.1.0",
     "babel-preset-shopify": "^21.0.0",
+    "css-loader": "3.4.2",
     "faker": "^4.1.0",
-    "preact": "^10.4.1",
+    "file-loader": "5.0.2",
+    "image-webpack-loader": "6.0.0",
+    "mini-css-extract-plugin": "0.9.0",
     "prettier": "^2.0.5",
-    "stylelint-config-shopify": "^7.4.0",
-    "stylelint-use-nesting": "^2.0.0"
+    "postcss-functions": "^3.0.0",
+    "postcss-loader": "^3.0.0",
+    "postcss-preset-env": "^6.7.0",
+    "style-loader": "1.1.3",
+    "stylelint-use-nesting": "^2.0.0",
+    "stylelint-config-shopify": "^7.4.0"
   },
   "peerDependencies": {
     "preact": "^10.4.1"
   },
   "dependencies": {
-    "@babel/preset-env": "^7.0.0",
-    "@quilted/react-auto-headings": "^0.1.0",
     "@shopify/argo-checkout": "^0.10.1",
     "@shopify/css-utilities": "1.0.5",
-    "@shopify/react-hooks": "1.9.1",
```

Comment:
We were depending on this, but never using it?￼

Reply:
Nope, there was no hits by quickly searching for the import name across the codebase.

Reply:
It's also possible that we used it in the past, but stopped and some point and simply missed to remove it from the package.json